src/vl-map-wms-layer.js

```diff
@@ -52,11 +51,10 @@ export class VlMapWmsLayer extends VlMapLayer {
   }
 
   get _sldBody() {
-    return this.getAttribute('data-vl-sld-body') ? this.__sldBodyWithWhitespaceRemoved : '';
-  }
-
-  get __sldBodyWithWhitespaceRemoved() {
-    return this.getAttribute('data-vl-sld-body').replace(/>\s*/g, '>').replace(/\s*</g, '<');
+    const wmsStyle = this.querySelector(':scope > vl-map-wms-style');
+    if (wmsStyle) {
+      return wmsStyle._sldBody;
+    }
   }
```

Comment:
denk dat, om het "gelijk te trekken" met de style voor WFS, de style zichzelf op de layer moet zetten, ipv dat de layer zijn styles moet gaan zoeken ...

Reply:
Had ik ook geprobeerd, maar het probleem hierbij was dat de layer nog moet toegevoegd worden, terwijl bij de VlMapVectorLayer die layer al is aangemaakt en de setStyle function wordt gebruikt. De parameter veranderen na de aanmaak kan ook wel met `updateParams`, maar mogelijk gaat er dan dubbel opgehaald worden (een keer met de oude stijl en dan een keer met de nieuwe).